package-lock.json

```diff
@@ -10313,17 +9307,28 @@
         "cross-fetch": "^3.1.5"
       }
     },
+    "packages/chat-core-aws-connect/node_modules/amazon-connect-chatjs": {
+      "version": "2.3.2",
```

Comment:
did some digging, and it seems like this update, 2.3.0 => 2.3.2, include a change that added some new deps for amazon-connect-chatjs (see [here](https://github.com/amazon-connect/amazon-connect-chatjs/commit/d60bc6f51757164cae09bf4e72c8c98adb8b4451#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R88-R91)). Although the new jest-dom dep really should be in devDeps.

Reply:
Thanks for looking into that! Makes sense to me